### DISCUSSION TITLE
Dislike bots?

### DISCUSSION THREAD

**Discussion by CrocodileCroco** [Anarios/return-youtube-dislike#724] [General]
I have seen videos with this extension enabled that have 1 million dislikes (against 900k likes)

That's very weird, i think this extension is being botted. Mostly on YouTube Shorts

Examples of videos having botted dislikes using this extension : 
- LGBT videos
- "Trend" videos
- etc..

That is not normal, i think there should be a way to stop these bots.

**Comment by cyrildtm** [comment]
This was predicted and expected to happen. See: https://github.com/Anarios/return-youtube-dislike/issues/344#issuecomment-1159312453


To be clear: I did not do it. It's technically feasible, but not worth my time. Also, I have no problems with LGBT or "trend" videos. Or etc.

**Comment by Anarios** [comment]
Links would be helpful. But considering that people who install the extension are much more likely to dislike something... It isn't surprising that "hated" videos will get even more hate.
